### DISCUSSION TITLE
Data on parents and offspring

### DISCUSSION THREAD

**Discussion by MWalbecq** [PasaOpasen/geneticalgorithm2-6.8.7#58] [Ideas]
First of all, I would like to thank you for this very complete library which works very well.

I am currently trying to form a graph on which I would like to represent the evolution of the population. The idea is to represent each individual of the population and to link them to their offspring until the end of the algorithm. I tried to use the `MiddleCallbacks` but I couldn't really access this data.

So I was wondering if there is a way to have data on which we have all the individuals created and their offspring ?

**Comment by PasaOpasen** [comment]
@MWalbecq It's to difficult to get this story, sorry

**Comment by PasaOpasen** [comment]
The algorithm is based on full population, it does not operate of individuals cuz of performance
